File: Contests/Div4/835/B.py
from string import ascii_lowercase
def solve():
    n = int(input())
    s = input()
    alf = ascii_lowercase
    maior = 0
    for i in s:
        maior = max(maior, alf.index(i))
    print(maior+1)


t = int(input())
for i in range(t):
    solve()

File: Contests/Div4/835/B.cpp
#include <bits/stdc++.h>

using namespace std;

void solve(){
    int n;
    cin >> n;
    string s;
    cin >> s;
    string alf = "abcdefghijklmnopqrstuvwxyz";
    int maior = 0;
    for (int i = 0; i < s.size(); i++){
        int a = alf.find(s[i]);
        maior = max(maior, a);
    }
    cout << maior+1 << '\n';
}

int main(void){
    ios_base::sync_with_stdio(false); cin.tie(0);
    int t;
    cin >> t;
    while (t--){
        solve();
    }
}